index.d.ts

```diff
@@ -1600,25 +1601,25 @@ declare module 'discord.js' {
 		maxID?: Snowflake;
 		minID?: Snowflake;
 		has?: 'link'
-			| 'embed'
```

Comment:
I'm not sure why you changed the indentations on these, but this should be reverted. I'll assume this was also your formatting tool. You should avoid using that when contributing to someone else's projects.

Reply:
Its done.